docs/index.md

```diff
@@ -2,12 +2,12 @@
 layout: "wallarm"
 page_title: "Provider: Wallarm"
 description: |-
-  The Wallarm provider is used to interact with the Wallarm WAF resources. The provider needs to be configured with the proper authentication credentials before it can be used.
+  The Wallarm provider is used to interact with the Wallarm API Security resources. The provider needs to be configured with the proper authentication credentials before it can be used.
```

Comment:
let's try to do not mention "Wallarm API Security" , use "the Wallarm platform/solution" instead since new subscription structure reflects API Security as one of the solution use cases but this provider is for the whole solution